pkg/ctl/create/nodegroup.go

```diff
@@ -168,8 +168,6 @@ func doCreateNodeGroups(cmd *cmdutils.Cmd, updateAuthConfigMap bool) error {
 				logger.Info("as you are using a GPU optimized instance type you will need to install NVIDIA Kubernetes device plugin.")
 				logger.Info("\t see the following page for instructions: https://github.com/NVIDIA/k8s-device-plugin")
 			}
-
-			return nil
```

Comment:
Ok, so this a stale line since we removed `ngFilter.ForEach`, right? In that case, I think this a candidate for 0.5.3.

Reply:
That's right. I was also planning to include it in 0.5.3.